src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CertificateAssetDownloader.cs

```diff
@@ -0,0 +1,127 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Internal.Cryptography.Pal
+{
+    internal static class CertificateAssetDownloader
+    {
+        private static unsafe Interop.libcurl.curl_unsafe_write_callback s_writeCallback = CurlWriteCallback;
+
+        static CertificateAssetDownloader()
+        {
+            Interop.libcurl.curl_global_init(Interop.libcurl.CurlGlobalFlags.CURL_GLOBAL_ALL);
```

Comment:
What happens if both this library and HttpClient call curl_global_init?  Eventually all of those will be called through our shims I believe (@nguerrera, correct?), but at least in the meantime do we need some sort of initialization mechanism like we have for OpenSSL?  Or can this be called any number of times and it's fine?

The documention for this is also pretty scary:
```
This function is not thread safe. You must not call it when any other thread in the program (i.e. a thread sharing the same memory) is running. This doesn't just mean no other thread that is using libcurl. Because curl_global_init calls functions of other libraries that are similarly thread unsafe, it could conflict with any other thread that uses these other libraries.
```